tools/migration-unified-annotation-versioning/count_skeleton_keys.py

```
@@ -0,0 +1,42 @@
+import logging
+from utils import setup_logging, log_since
+import argparse
+from connections import connect_to_fossildb, connect_to_postgres, assert_grpc_success
```

Ruff (0.8.2)

`connections.connect_to_postgres` imported but unused

Remove unused import: `connections.connect_to_postgres`

(F401)